README.md

```diff
@@ -1,291 +1,124 @@
 <a name="readme-top"></a>
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍